content/en/examples/pods/pod-configmap-envFrom.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: dapi-test-pod
+  name: my-pod-with-env-configmap
+  labels:
+    name: use-configmap-env
 spec:
   containers:
-    - name: test-container
-      image: k8s.gcr.io/busybox
-      command: [ "/bin/sh", "-c", "env" ]
-      envFrom:
-      - configMapRef:
-          name: special-config
-  restartPolicy: Never
+  - envFrom:
+    -configMapRef:
+	       name: my-config-map-name
```

Comment:
I'm not sure why there's this red sign here. Maybe it's because of some sort of indentation error?

Reply:
Hi @Aut0R3V . The red block is most likely a tab. You can remove the tab (or extra spaces) in your editor.

Reply:
Thanks!